sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobClientBase.java

```diff
@@ -412,9 +414,12 @@ public void download(OutputStream stream) {
     public BlobDownloadResponse downloadWithResponse(OutputStream stream, BlobRange range,
         DownloadRetryOptions options, BlobRequestConditions requestConditions, boolean getRangeContentMd5,
         Duration timeout, Context context) {
+        Context updatedContext = context == null ? Context.NONE.addData(CoreConstants.DISABLE_BUFFER_COPY, true)
+            : context.addData(CoreConstants.DISABLE_BUFFER_COPY, true);
```

Comment:
I'd like to see this kind of repetitive (and verbose) code moved into a utility method.